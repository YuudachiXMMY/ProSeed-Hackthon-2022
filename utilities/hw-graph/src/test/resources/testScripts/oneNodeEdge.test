# Test to adding a node with only one edge, which points to the node itself

# Create a graph
CreateGraph graph1

# Add a node to graph
AddNode graph1 node1

# Add an edge to graph
AddEdge graph1 node1 node1 edge1

# Print all nodes in graph
ListNodes graph1
ListChildren graph1 node1