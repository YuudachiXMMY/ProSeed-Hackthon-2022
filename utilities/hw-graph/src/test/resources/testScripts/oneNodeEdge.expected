# Test to adding a node with only one edge, which points to the node itself

# Create a graph
created graph graph1

# Add a node to graph
added node node1 to graph1

# Add an edge to graph
added edge edge1 from node1 to node1 in graph1

# Print all nodes in graph
graph1 contains: node1
the children of node1 in graph1 are: node1(edge1)